packages/engine/Source/Renderer/Context.js

```diff
@@ -42,9 +43,18 @@ function Context(canvas, options) {
   Check.defined("canvas", canvas);
   //>>includeEnd('debug');
 
+  let requestWebgl2 = false;
+  if (defined(options) && defined(options.requestWebgl2)) {
+    requestWebgl2 = options.requestWebgl2;
+    Context._deprecationWarning(
+      "ContextOptions.requestWebgl2",
+      "Cesium.requestWebgl2 was deprecated in CesiumJS 1.101 and will be removed in 1.102. Use Cesium.requestWebgl1 to request a WebGL1 or WebGL2 context."
```

Comment:
```suggestion
      "Cesium.requestWebgl2 was deprecated in CesiumJS 1.101 and will be removed in 1.102. Instead, CesiumJS will default to using a WebGL2 context for rendering. Use Cesium.requestWebgl1 to request a WebGL1 or WebGL2 context."
```

How about this, just to provide a bit more guidance if a user sees this message.

Reply:
That's better, updated.